File: fizzbuzz.php
<?php

        foreach (range(0, 100) as $number)
        {
                $output = '';

                if(($number % 3 == 0) && ($number % 5 == 0))
                {
                        $output = 'FizzBuzz';
                }

                elseif($number % 3 == 0)
                {
                        $output = 'Fizz';
                }

                elseif($number % 5 == 0)
                {
                        $output = 'Buzz';
                }

                else
                {
                        $output = $number;
                }


                print "$output\n";
        }
?>

File: fizzbuzz.py
#!/usr/bin/python -tt

def main():

        for i in range(100):

                if i%3 == 0 and i%5 == 0:
                        output = 'FizzBuzz'
                elif i%3 == 0:
                        output = 'Fizz'
                elif i%5 == 0:
                        output = 'Buzz'
                else:
                        output = str(i)

                print output

if __name__ == '__main__':
        main()
